Magento2/Sniffs/Functions/FunctionsDeprecatedWithoutArgumentSniff.php

```diff
@@ -31,15 +31,15 @@ class FunctionsDeprecatedWithoutArgumentSniff implements Sniff
     private const WARNING_CODE = 'FunctionsDeprecatedWithoutArgument';
 
     /**
-     * Deprecated functions without argument.
+     * Deprecated functions without argument
      *
      * @var array
      */
-    private const FUNCTIONS_LIST = [
-        'mb_check_encoding',
-        'get_class',
-        'get_parent_class',
-        'get_called_class'
+    private const DEPRECATED_FUNCTIONS_AND_FIXES = [
+        'mb_check_encoding' => 'null',
```

Comment:
`mb_check_encoding(null);` results in the following error `Deprecated: mb_check_encoding(): Calling mb_check_encoding() without argument is deprecated in /in/HGVsB on line 8` please correct the argument

Reply:
check the latest commits, please